### PR TITLE
make marketial message box look good with no header/image components …

### PR DESCRIPTION
**Pull request by zivl**
There is no change in API, props or functionality, just styling is now applied only if there is a given header. This prevents the empty white block when no header/image components are passed.

please take a look at the following slack post:
https://wix.slack.com/archives/C39FTGUBZ/p1574681619499900

### ✅ Checklist

- [X] 👨‍🎨 UX change is approved by the Design System UX  - tagged
- 📚 Change is documented
  - [X] Story
- 🔬 Change is tested
  - [X] Component

### CODE REVIEW

File: src/MessageBox/MarketerialLayout/MessageBoxMarketerialLayout.scss [hunk 1]
```diff
@@ -7,13 +7,20 @@
   background-color: #fff;
   border-radius: 8px;
 }
+
+$no-header-height: 90px;
```
Comment (TalZa1810) [inline src/MessageBox/MarketerialLayout/MessageBoxMarketerialLayout.scss]: $noHeaderHeight: 54px;

**Comment by TalZa1810** [comment]
Moreover, can you please add visual tests to the following files:

https://github.com/wix/wix-style-react/blob/88102cd46ec1d8acd1e4124bace98fbe6d56d5fa/src/MessageBox/MarketerialLayout/MarketrialLayout.visual.js#L16

https://github.com/wix/wix-style-react/blob/88102cd46ec1d8acd1e4124bace98fbe6d56d5fa/src/MessageBox/test/MessageBox.visual.js#L21

Thanks :)

File: src/MessageBox/MarketerialLayout/MessageBoxMarketerialLayout.scss (hunk 1 above)
Comment (zivl) [reply src/MessageBox/MarketerialLayout/MessageBoxMarketerialLayout.scss]: done

File: src/MessageBox/MarketerialLayout/MarketrialLayout.visual.js [hunk 2]
```diff
@@ -38,6 +38,14 @@ const tests = [
           ...baseProps,
         },
       },
+      {
+        it: 'Without header',
+        props: {
+          title: 'No Header',
+          content: 'Some content to show you',
+          theme: 'white',
```
Comment (TalZa1810) [inline src/MessageBox/MarketerialLayout/MarketrialLayout.visual.js]: This is not a real scenario. Please add also these props:
```
primaryButtonLabel: "Primary Button"
primaryButtonTheme: 'blue'
```

Comment (zivl) [reply src/MessageBox/MarketerialLayout/MarketrialLayout.visual.js]: why this is not real scenario? 
this is my scenario. The close button above is enough for me.
I'm not obligated to set a CTA button

Comment (TalZa1810) [reply src/MessageBox/MarketerialLayout/MarketrialLayout.visual.js]: The `primaryButtonLabel` is required:
https://wix-style-react.now.sh/?activeTab=API&addons=0&full=0&panelRight=0&selectedKind=9.%20Modals&selectedStory=9.4%20Announcement&stories=1

**Pull request merged**